src/shared/components/create/NoteCreator.tsx

```diff
@@ -25,7 +25,7 @@ interface NoteCreatorProps {
   reset?: boolean
 }
 
-function addPTags(event: NDKEvent, repliedEvent?: NDKEvent, quotedEvent?: NDKEvent) {
+function addTags(event: NDKEvent, repliedEvent?: NDKEvent, quotedEvent?: NDKEvent) {
```

Comment:
Renamed `addPTags` to `addTags` because it was doing more than just adding p-tags.